docs/Tech_grant_model_fairness.md

```diff
@@ -0,0 +1,132 @@
+e.g. for the Recipient organisation grouping and F1 metric this would be
+ ```
+ max('Golden triangle F1', 'Not golden triangle F1') - min('Golden triangle F1', 'Not golden triangle F1')
+ ```
+
+| Model name 	| Grouping 	| f1 difference 	| precision_score difference	| recall_score difference	|
+|-	|-	|-	|-	|-	|
+| Ensemble predictions - 2 models 	| Recipient organisation 	| 0.011 	| 0.071 	| 0.058 	|
+|  	| Region 	| 0.020 	| 0.081 	| 0.135 	|
+| Ensemble predictions - 3 models 	| Recipient organisation 	| 0.001 	| 0.036 	| 0.034 	|
+|  	| Region 	| 0.022 	| 0.048 	| 0.002 	|
+| bert_SVM_bert_201022 predictions 	| Recipient organisation 	| 0.038 	| 0.067 	| 0.003 	|
+|  	| Region 	| 0.012 	| 0.082 	| 0.081 	|
+| bert_SVM_scibert_201022 predictions 	| Recipient organisation 	| 0.021 	| 0.023 	| 0.019 	|
+|  	| Region 	| 0.021 	| 0.093 	| 0.123 	|
+
+We can visualise the precision (lighter) and recall (darker) differences for each model and grouping (reds - recipient organisation, blues - region):
+
+![](diff_precrec.jpeg)
+
+## Selecting the best model
+
+As mentioned earlier we want to pick a model with both the best test scores and the best fairness scores.
+
+To find the best overall fairness score, we added the metric differences for each grouping, e.g. if the F1 difference for the recipient organisation grouping was 0.011 and for the region grouping it is 0.02, then we add these to get 0.031.
+
+The lowest scores for all three of the metrics are found in the **'Ensemble predictions - 3 models'** model.
+
+| Model name 	| f1 sum difference	| precision_score sum difference | recall_score sum difference	|
+|-	|-	|-	|-	|
+| Ensemble predictions - 3 models 	| **0.024** 	| **0.083** 	| **0.035** 	|
+| Ensemble predictions - 2 models 	| 0.031 	| 0.152 	| 0.194 	|
+| bert_SVM_scibert_201022 predictions 	| 0.042 	| 0.116 	| 0.142 	|
+| bert_SVM_bert_201022 predictions 	| 0.050 	| 0.150 	| 0.084 	|
+
+And a recap of the test results shows this model also gives relatively high across all metrics:
+
+| Model name   	| f1 	| precision_score 	| recall_score 	|
+|-	|-	|-	|-	|
+| Ensemble predictions - 2 models| 0.885 	| 0.862 	| 0.909 	|
+| Ensemble predictions - 3 models| **0.873** 	| **0.873** 	| **0.873** 	|
+| bert_SVM_scibert_201022 predictions| 0.879 	| 0.904 	| 0.855 	|
+| bert_SVM_bert_201022 predictions| 0.881 	| 0.825 	| 0.945 	|
+
+Thus we select this as our best model.
+
+## Fairness results of best model
+
+And a recap of all the fairness results for this model:
+
+|	Grouping | Group |	Sample size	accuracy	|f1	|precision_score|	recall_score|
+|---|---|---|---|---|---|
+| Recipient organisation	|Golden triangle	|41	|0.872	|0.850|	0.895|
+|Recipient organisation	|Not golden triangle|	66		|0.873	|0.886|	0.861|
+|Region grouped	|Greater London	|30		|0.867|	0.867|	0.867|
+|Region grouped	|International	|10|		0.750	|0.600	|1.000|
+|Region grouped	|UK, not greater London	|67	|0.889	|0.914	|0.865|
+
+## Conclusion:
```

Comment:
I think this section is more like summary of findings, rather than conclusions, as I don't see what was concluded upon, you know? A conclusion would like "this model is most fair according to our definition and interpretation, which would probably need to be a team discussion anyway, so would be good for each of us to input to that section through the PR at least.